melos.yaml

```diff
@@ -186,8 +186,9 @@ scripts:
 
   generate:pigeon:
     run: |
-      melos exec -- "flutter pub run pigeon --input ./pigeons/messages.dart"
-      melos run generate:pigeon:macos
+      melos exec -- "flutter pub run pigeon --input ./pigeons/messages.dart" && \
```

Comment:
Is that &&\ needed? It wasn't there before

Reply:
Without it, the second command would run even if the first one fails. 
If there is an issue with the generation, it's better to not run the other commands IMO